drive/src/drive/cache.rs

```diff
@@ -0,0 +1,28 @@
+use crate::contract::Contract;
+use crate::drive::flags::StorageFlags;
+use crate::fee::FeeResult;
+use costs::OperationCost;
+use moka::sync::Cache;
+use std::sync::Arc;
+
+/// Drive cache struct
+pub struct DriveCache {
```

Comment:
I think it's a bad idea to combine all caches together in this case you need to pass to the contract module the whole thing including other caches.

Reply:
I just moved it to it's own file.